### PR TITLE
⌗117795 TEC and ET Welcome Page Refreshes

### PR DESCRIPTION
**Pull request by ggwicz**
**Ticket:** [**⌗117795**](http://central.tri.be/issues/117795)

---

This PR is of a piece with these other two PRs:
* https://github.com/moderntribe/tribe-common/pull/890
* https://github.com/moderntribe/event-tickets/pull/1016

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:11](https://github.com/moderntribe/the-events-calendar/blob/ebcd5f47e433862a632c0aaace04fa7eed142048/src/admin-views/admin-welcome-message.php#L11)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:11](https://github.com/moderntribe/the-events-calendar/blob/4c8d52ad39087f7187a027d2beb13f7a2e941628/src/admin-views/admin-welcome-message.php#L11)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/f23f717e384e346dee538685c88e87f8fa2c8332/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/92e994d50793598e95602b0dee55eadd44539477/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/96cdb4a7e4574de4d2ef412627f92723349e6286/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -951,9 +951,9 @@ public function activation_page() {
 					'version'               => self::VERSION,
 					'plugin_path'           => $this->plugin_dir . 'the-events-calendar.php',
 					'version_history_slug'  => 'previous_ecp_versions',
-					'update_page_title'    => __( 'Welcome to The Events Calendar', 'the-events-calendar' ),
-					'update_page_template' => $this->plugin_path . 'src/admin-views/admin-update-message.php',
-					'welcome_page_title'    => __( 'Welcome to The Events Calendar', 'the-events-calendar' ),
+					'update_page_title'     => __( 'Welcome to The Events Calendar', 'the-events-calendar' ),
+					'update_page_template'  => $this->plugin_path . 'src/admin-views/admin-update-message.php',
+					'welcome_page_title'    => __( 'Welcome to The Events Calendar!', 'the-events-calendar' ),
```
Comment (Camwyn) [inline src/Tribe/Main.php]: Added ❗️ here but not in `update_page_title` above?

File: src/admin-views/admin-welcome-message.php [hunk 2]
```diff
@@ -1,56 +1,73 @@
...
 	<div class="tribe-half-column">
-		<h4><?php esc_html_e( 'Getting Started', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/no" target="_blank"><?php esc_html_e( 'Check out the New User Primer &amp; Tutorials', 'the-events-calendar' ); ?></a></p>
-
-		<h4><?php esc_html_e( 'Looking for More Features?', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/np" target="_blank"><?php esc_html_e( 'Addons for Community, Tickets, Filters, and more.', 'the-events-calendar' ); ?></a></p>
-
-		<h4><?php esc_html_e( 'Support Resources', 'the-events-calendar' ); ?></h4>
-		<p><a href="https://m.tri.be/nq" target="_blank"><?php esc_html_e( 'FAQs, Documentation, Tutorials and Forums', 'the-events-calendar' ); ?></a></p>
+		<h2><?php esc_html_e( 'The Latest and Greatest', 'the-events-calendar' ); ?></h2>
```
Comment (Camwyn) [inline src/admin-views/admin-welcome-message.php]: There is a lot of switching from `<h4>` to `<h2>` through here (and in the other PRs?) - is this format/size or context/syntactical?

Comment (ggwicz) [reply src/admin-views/admin-welcome-message.php]: Mainly for formatting/size

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/d2e4f6f29223b1226f8c787060436bd409c7fa14/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/59b931d0e2aedd30d645ff275d6bc064d79a44c9/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
Why is this pending @ggwicz, are we missing anything?

**Comment by bordoni** [comment]
I had to punt it to next Release due to this not been merged.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.19*, but this was pull requested against *release/M19.01*.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:11](https://github.com/moderntribe/the-events-calendar/blob/59b931d0e2aedd30d645ff275d6bc064d79a44c9/src/admin-views/admin-welcome-message.php#L11)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [admin-welcome-message.php:14](https://github.com/moderntribe/the-events-calendar/blob/59b931d0e2aedd30d645ff275d6bc064d79a44c9/src/admin-views/admin-welcome-message.php#L14)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not 'Tribe__Events__Main' | [admin-welcome-message.php:7](https://github.com/moderntribe/the-events-calendar/blob/59b931d0e2aedd30d645ff275d6bc064d79a44c9/src/admin-views/admin-welcome-message.php#L7)

> via jenkins/codesniffer-phpcs

**Pull request closed without merging**